src/Markdig.Tests/TestCst.cs

```diff
@@ -0,0 +1,673 @@
+using Markdig.Renderers;
+using Markdig.Renderers.Normalize;
+using Markdig.Syntax;
+using NUnit.Framework;
+using System.IO;
+
+/// <summary>
+/// General notes
+/// - whitespace can occur before, between and after symbols
+/// </summary>
+/// TODO:
+/// - \r\n, \r, \n
+/// - \t and spaces
+/// - html entities i.e. &gt;
+/// 
+namespace Markdig.Tests
+{
+    [TestFixture]
+    public class TestCst
```

Comment:
Don't use the term Cst. Prefer something like Roundtrip

Reply:
done